x-pack/test/detection_engine_api_integration/security_and_spaces/group10/perform_bulk_action.ts

```diff
@@ -1898,13 +1936,20 @@ export default ({ getService }: FtrProviderContext): void => {
           })
           .expect(200);
 
-        expect(setIndexBody.attributes.summary).to.eql({ failed: 0, succeeded: 1, total: 1 });
+        expect(setIndexBody.attributes.summary).to.eql({
+          failed: 0,
+          skipped: 1,
+          succeeded: 0,
+          total: 1,
+        });
 
-        // Check that the updated rule is returned with the response
-        expect(setIndexBody.attributes.results.updated[0].index).to.eql(undefined);
-        expect(setIndexBody.attributes.results.updated[0].data_view_id).to.eql(dataViewId);
```

Comment:
We should make sure that these were actually not modified, so maybe we could fetch the rule and assert its attributes.

Reply:
Done